### DISCUSSION TITLE
我，琪亚娜，打钱

### DISCUSSION THREAD

**Discussion by Demise-deity** [share121/inter-knot#743] [其他]
偷偷告诉你我其实是终焉之神，但因为特殊原因失去力量掉到了这个世界里，只要你v我50000丁尼助力我回到本征世界，我将赐予你匹敌律者的力量
![image](https://github.com/user-attachments/assets/e5bdb348-1d0d-4d61-a32a-b5d88ce964c6)

**Comment by poikl709** [comment]
111真的吗，送233巴普洛晶核行吗

**Comment by Demise-deity** [comment]
只要你帮助我，甭管233巴普洛晶核，直接2333记忆序列

**Reply by GanlinSama** [reply]
卡**丘，启动！

**Reply by poikl709** [reply]
琪亚娜万岁！！！

**Comment by GanlinSama** [comment]
琪亚娜穿好衣服啊啊啊啊啊w(ﾟДﾟ)w

**Comment by KSN-tt41** [comment]
1111111琪天大圣送终焉之茧吗

**Reply by Demise-deity** [reply]
送你一把薪炎王剑

**Reply by abandoned-person** [reply]
诶嘿

**Comment by Callkit-P0p** [comment]
111可以再送一个你的等身抱枕吗?

**Reply by Demise-deity** [reply]
什么什么，你在想什么

**Comment by keerdun** [comment]
1111可以送煮饭婆吗

**Reply by Demise-deity** [reply]
你怎么能抢我老婆

**Comment by aaa00073** [comment]
11111111111111111111111这么可爱我愿意被她骗！[口水]

**Comment by Elysia041** [comment]
@雷电芽衣

**Comment by poikl8i** [comment]
11111大哥真送吗

**Comment by Kiana175** [comment]
别信这家伙，她要是琪亚娜，那本小姐是什么？
